src/Directory.Build.targets

```diff
@@ -1,3 +1,9 @@
 <Project>
   <Import Project="Packaging.targets" Condition=" '$(IsPackable)' == 'true' " />
+
+  <Target Name="_SetVSTestLogger" BeforeTargets="VSTest" Condition=" '$(TrxLogFileNameSuffix)' != '' ">
+  <PropertyGroup>
+    <VSTestLogger>trx%3BLogFileName=$(AssemblyName)-$(TrxLogFileNameSuffix)-$(TargetFramework).trx</VSTestLogger>
```

Comment:
This is the fix for:
![image](https://github.com/user-attachments/assets/6d551cd2-cc84-4b36-acad-973037498e91)